x-pack/plugins/observability_solution/entity_manager/server/lib/entities/install_entity_definition.ts

```diff
@@ -73,6 +75,10 @@ export async function installEntityDefinition({
     logger.debug(`Installing transforms for definition ${definition.id}`);
     await createAndInstallHistoryTransform(esClient, entityDefinition, logger);
     installState.transforms.history = true;
+    if (entityDefinition.history.settings?.backfillSyncDelay) {
+      await createAndInstallHistoryBackfillTransform(esClient, entityDefinition, logger);
```

Comment:
Should also uninstall it

Reply:
Yes... I haven't gotten that far yet 😄, it's still a draft.